registry/settings.py

```diff
@@ -16,6 +15,6 @@
 SQLALCHEMY_DATABASE_URI = env.str("DATABASE_URL")
 SECRET_KEY = env.str("SECRET_KEY")
 BCRYPT_LOG_ROUNDS = env.int("BCRYPT_LOG_ROUNDS", default=13)
-DEBUG_TB_ENABLED = DEBUG
```

Comment:
Vážně v rámci tohoto PR? Pokud to nechceš používat, můžeme to navázat na obsah `.env` souboru, aby sis to mohl vypnout.

Reply:
To byl omyl.

Reply:
Omlouvám se, jestli jsem reagoval přehnaně, ale vzpomněl jsem si, že sis na debug panel stěžoval a napadlo mě, že ses jeho vypnutí pokusil propašovat tudy :wink: